### PR TITLE
xmlJson: add XML/JSON translator app

### PR DESCRIPTION
**Pull request by davidkostov**
Added a JSON or XML translating application.

### CODE REVIEW

File: src/test/java/com/clouway/translators/JsonTranslatorTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        catch(IOException ioe){
+            ioe.printStackTrace();
+        }
+    }
+
+    @Test
+    public void objectToJsonStringThanReverseAndUseObject(){
+        try{
+            String jsonString = translator.toJsonString(new Person(29, "Bob"));
+            System.out.println(jsonString);
```
Comment (mgenov) [inline src/test/java/com/clouway/translators/JsonTranslatorTest.java]: System.out could be removed.

File: src/test/java/com/clouway/translators/JsonTranslatorTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        try{
+            String jsonString = "[{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}," +
+                    "{\"age\":29,\"name\":\"Bob\"}]\n";
+
+            List<Person> people = translator.toObjectFromString(jsonString,new TypeReference<List<Person>>(){});
```
Comment (mgenov) [inline src/test/java/com/clouway/translators/JsonTranslatorTest.java]: Code is not formatted.

File: src/test/java/com/clouway/translators/XmlTranslatorTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        catch(IOException ioe){
+            ioe.printStackTrace();
+        }
+    }
+
+    @Test
+    public void objectToXmlStringThanReverseAndUseObject(){
+
+        String xmlString = translator.toXmlString(new Person(29, "Bob"));
+        System.out.println(xmlString);
```
Comment (mgenov) [inline src/test/java/com/clouway/translators/XmlTranslatorTest.java]: System.out statements should be omitted from tests.

File: src/test/java/com/clouway/translators/XmlTranslatorTest.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        Person person = translator.toObjectFromString(xmlString);
+
+        assertThat(person.getName(), equalTo("Bob"));
+
+    }
+
+    @Test
+    public void xmlStringToObject(){
+
+        String xmlString = "<com.clouway.main.Person>\n" +
```
Comment (mgenov) [inline src/test/java/com/clouway/translators/XmlTranslatorTest.java]: You can map class type to appropriate tag name and XML will look:

```xml
<Person>
   <age>29</age>
   <name>Bob</name>
</Person>
```

which is more reasonable as name.

**Comment by davidkostov** [comment]
Removed System.out-s, formatted code, made an alias method for the xml translator so that aliases can be set for the tags of the serialized objects.

**Pull request merged**